core/src/main/scala/kanaloa/reactive/dispatcher/queue/QueueProcessor.scala

```diff
@@ -116,11 +118,10 @@ trait QueueProcessor extends Actor with ActorLogging with MessageScheduler {
     case _ ⇒ //Ignore
   }
 
-  private def createWorker(index: Int): WorkerRef = {
-    val timestamp = LocalDateTime.now.toInstant(ZoneOffset.UTC).toEpochMilli
+  private def createWorker(): WorkerRef = {
     val worker = context.actorOf(
       workerProp(queue),
-      s"worker-${queue.path.name}-$index-${timestamp}"
+      s"worker-${workerCount.incrementAndGet()}"
```

Comment:
This is the name of the `Worker` which has to be unique within a QueueProcessor. Not sure I follow the relationship between this and Worker's `receive`